_data/meltano/utilities/airflow/meltano.yml

```diff
@@ -0,0 +1,81 @@
+name: airflow
+namespace: airflow
+label: Airflow
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/meltano/airflow-ext
```

Comment:
Let's keep this as the apache repo. I don't want people to come to https://deploy-preview-822--meltano-hub.netlify.app/utilities/airflow and walk away thinking Meltano maintains its own fork of Airflow. It should be clear this is the full Airflow tool and we've got an extension that makes it easy to use with Meltano. 

I think it'd be worth it to add a new key for `ext_repo`. In a separate PR we (not you) can get the frontend updated to render an extension link and maybe even an EDK badge.